common/src/main/scala/explore/model/reusability.scala

```diff
@@ -70,4 +71,6 @@ object reusability {
   implicit val catalogTargetResultReuse: Reusability[CatalogTargetResult]          = Reusability.derive
   implicit val scienceConfigurationnResultReuse: Reusability[ScienceConfiguration] =
     Reusability.byEq
+
+  implicit val existenceReuse: Reusability[Existence] = Reusability.derive
```

Comment:
I'm not sure if there is a better place for this?

Reply:
This is probably the right place